GoogleTestAdapter/Core/Runners/SequentialTestRunner.cs

```diff
@@ -137,12 +138,12 @@ public void Cancel()
 
         public static void LogExecutionError(ILogger logger, string executable, string workingDir, string arguments, Exception exception, string threadName = "")
         {
-            logger.LogError($"{threadName}Failed to run test executable '{executable}': {exception.Message}");
-            logger.DebugError($@"{threadName}Stacktrace:{Environment.NewLine}{exception.StackTrace}");
+            logger.LogError(String.Format(Resources.RunExecutableError, threadName, executable, exception.Message));
+            logger.DebugError(String.Format(Resources.StackTrace, threadName, Environment.NewLine, exception.StackTrace));
             logger.LogError(
                 $"{threadName}{Strings.Instance.TroubleShootingLink}");
-            logger.LogError(
-                $"{threadName}In particular: launch command prompt, change into directory '{workingDir}', and execute the following command to make sure your tests can be run in general.{Environment.NewLine}{executable} {arguments}");
+            logger.LogError(String.Format(Resources.ExecuteSteps, threadName, workingDir) + $"{Environment.NewLine}{executable} {arguments}");
```

Comment:
This line and 142 are quite inconsistent in that in 142 NewLine and StackTrace as passed as fill-ins to the localized string, and in 145 NewLine and executeable, arguments are concatenated with the localized string.